users/models.py

```diff
@@ -147,15 +147,10 @@ def email_user(self, subject: str, message: str, from_email: str = None) -> None
 
     def activate_user(self) -> None:
         """
-        Activate and create a CustomUserPreferences object for the user.
-        Should be used only as initial activation for newly creating users.
+        Activate user.
```

Comment:
You can delete this docs string completely

Reply:
Done